hack/install_golangci.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+$GOBIN/golangci-lint --version | grep $VERSION
```

Comment:
Suggest validating at least `$VERSION` is non-empty.  Did you intended `grep -q`?  Also should probably `set -eo pipefail`.  Otherwise if `golangci-lint` dies, you'll miss it.

Reply:
Great catch! I'll open a follow-up PR.

Reply:
https://github.com/containers/libpod/pull/4879